package.json

```diff
@@ -16,17 +16,24 @@
   "private": true,
   "scripts": {
     "lint": "eslint src --ext .js",
+    "test": "jest --testPathPattern='/src/*'",
```

Comment:
As you created a `jest.config.js` you could maybe also put the testPathPattern there to have it in one place

Reply:
That's true, I've defined `testMatch` in the config file.